models/optional_modules/core_web_vitals/snowplow_web_vitals.sql

```diff
@@ -0,0 +1,32 @@
+{{
+  config(
+    materialized= 'incremental',
+    enabled=var("snowplow__enable_cwv", false),
+    unique_key='page_view_id',
+    upsert_date_key='derived_tstamp',
+    sort='derived_tstamp',
+    dist='page_view_id',
```

Comment:
There's no reason to not have these here, but they will only be relevant once we add redshift/postgres I think?

Reply:
yeah I know, they should come soon I hope